apps/scan/backend/src/scanners/custom/state_machine.ts

```diff
@@ -389,14 +395,17 @@ async function recordAcceptedSheet(
     // Gets reset to false within exportCastVoteRecordsToUsbDrive
     store.setIsContinuousExportOperationInProgress(true);
   });
-  (
-    await exportCastVoteRecordsToUsbDrive(
+
+  const exportResult = await continuousExportMutex.withLock(() =>
```

Comment:
To test this, it feels like we'd want to call `recordAcceptedSheet` two or more times without awaiting the results, then verify that everything is sane after we've awaited them all in batch.

Reply:
Yup exactly what I was going for